packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -473,7 +473,7 @@ class InteractiveGraphEditor extends React.Component<Props> {
                     </LabeledRow>
                 )}
                 {this.props.graph?.type &&
-                    this.props.apiOptions?.flags?.mafs?.["start-coords-ui"] && (
+                    (this.props.apiOptions?.flags?.mafs?.["start-coords-ui"][this.props.graph.type]) && (
```

Comment:
I think this needs to be

```suggestion
                    (this.props.apiOptions?.flags?.mafs?.["start-coords-ui"]?.[this.props.graph.type]) && (
```

or it will throw an error when `mafs["start-coords-ui"]` is undefined.